SonarQube.Common/AnalysisProperties/FilePropertyProvider.cs

```diff
@@ -64,7 +66,7 @@ public static bool TryCreateProvider(IEnumerable<ArgumentInstance> commandLineAr
                 }
                 else
                 {
-                    provider = new FilePropertyProvider(locatedPropertiesFile);
+                    provider = new FilePropertyProvider(locatedPropertiesFile, !settingsFileArgExists);
```

Comment:
If we have got a valid file, but the /s: argument wasn't supplied, then we must be using the default settings file.